src/utils/makeReactNativeConfig.js

```diff
@@ -178,6 +177,16 @@ const getDefaultConfig = ({
           : []
       ),
     resolve: {
+      alias:
+        process.env.NODE_ENV === 'production'
+          ? {}
+          : {
+              /**
+               * Latest `react-proxy` version does not contain try/catches from
+               * commit 981815dca250373619138c9f5aadf12295cf1b3f.
+               */
+              'react-proxy': '@zamotany/react-proxy',
```

Comment:
Maybe better to just use the GH url as a dependency rather than adding an alias cc @grabbou

Reply:
Using GH url won't work. `react-proxy` is required by `react-hot-loader` and have it as a dependency. Installing `react-proxy#981815dca250373619138c9f5aadf12295cf1b3f ` won't fix anything since `react-hot-loader` will have it's own version installed. That's why we need to alias it.

Reply:
Also, installing from commit, might work might not depending if yarn and npm will dedupe the package since version will be the same, but one will be from npm one from GH. Also, we would need to transpile `react-proxy` by outselves.

Reply:
Since `react-proxy` and `react-hot-loader` are not actively maintained, I'd say it's just simpler to have a working fork and just replace it later, when they fix issues / publish changes in the upstream